index.html

```diff
@@ -874,7 +869,33 @@
             <code id="idl-def-PushPermissionStatus.default">default</code>
           </td>
           <td>
-            The <a>webapp</a> needs to ask for permission in order to use Push API.
+            The <a>webapp</a> needs to ask for permission in order to use the Push API.
+          </td>
+        </tr>
+      </table>
+    </section>
+    <section>
+      <h2>
+        Exceptions
+      </h2>
+      <p>
+        The Push API uses the following new <a>DOMException</a> names.
```

Comment:
Should we maintain a reference to the bug (maybe even as an issue) to avoid two specifications mentioning they're about to introduce the PermissionDeniedError?